src/Domain/POV/ConfigCreator.php

```diff
@@ -158,6 +164,87 @@ public function create(
         return $dir;
     }
 
+    /**
+     * @throws Exception
+     */
+    private function getSELHeatmapRange(string $layerName): ?array
```

Comment:
so in the config file there is "SEL" section that has "heatmap_settings" for each SEL layer . E.g. for shipping intensity layers there is a "heatmap_range" which is an array which holds input and output data such that we can match the raster value to the actual shipping intensity , e.g. up to 500 ships hours/km2/month